### ISSUE TITLE
"url is not a function" error

### ISSUE THREAD

**Issue by dbhkhk** [bettiolo/oauth-signature-js#18]
Hi, bettiolo!
In the example in README, url is defined as a string, however, in the library it is defined as a function. When I used the code from the example, the error "url is not a function" occured. Maybe you can give it a new name instead of just "url" in the example in README.

Best,
Brody

**Comment by bettiolo** [comment]
Can you share some example code with me?

**Comment by dbhkhk** [comment]
Yes, [here](https://github.com/dbhkhk/neighborhood-map/commit/9186fd21f75818a99f7d42eaeceea5a084b3a330) you can see that I changed `url` to `yelpURL`. Before I made that change, the devtool console showed "url is not a function".

**Comment by nunoarruda** [comment]
Same error here....

**Comment by bettiolo** [comment]
@nunoarruda thank you for the feedback, I will have a look on this ASAP.

**Comment by gabrielAnglada** [comment]
Hi! In my case, I was using cloudboost framework and it has a bug which I have just reported (https://github.com/CloudBoost/JavaScriptSDK/issues/18)

Fixing it, the error dissapears.

Hope it helps

**Comment by bettiolo** [comment]
Thank you @gabrielAnglada

**Comment by springpriyanshi** [comment]
Hi team,

I am still getting this error with simple integration.var httpMethod = 'GET',
    url = 'using rest_url',
    parameters = {
        oauth_consumer_key : oauth_consumer_key,
        oauth_token : oauth_token,
        oauth_nonce : oauth_nonce,
        oauth_timestamp : oauth_timestamp,
        oauth_signature_method : 'HMAC-SHA1',
        oauth_version : oauth_version       
    },
    consumerSecret = oauth_consumer_secret,
    tokenSecret = oauth_token_secret,
	 // generates a RFC 3986 encoded, BASE64 encoded HMAC-SHA1 hash
    encodedSignature = oauthSignature.generate(httpMethod, url, parameters, consumerSecret, tokenSecret),
    // generates a BASE64 encode HMAC-SHA1 hash
    signature = oauthSignature.generate(httpMethod, url, parameters, consumerSecret, tokenSecret,
        { encodeSignature: false});
   Please suggest

**Comment by springpriyanshi** [comment]
Done guys just saw yelpurl changes Thanks
